sdk/eventhub/event-hubs/src/models/public.ts

```diff
@@ -128,6 +128,61 @@ export interface EventHubClientOptions {
   userAgent?: string;
 }
 
+/**
+ * Describes the options that can be provided while creating the EventHubConsumerClient.
+ */
+export interface EventHubConsumerClientOptions extends EventHubClientOptions {
+  /**
+   * An options bag to configure load balancing settings.
```

Comment:
are the option comments necessary or can we comment what each option does in the doc comments for `LoadBalancingOptions`?

Reply:
These comments should be on the EventHubConsumerClientOptions interface, not the loadBalancingOptions field, will fix!

Reply:
Updated!